spec/02-integration/05-proxy/02-router_spec.lua

```diff
@@ -235,6 +235,34 @@ for _, strategy in helpers.each_strategy() do
             hosts     = { "serviceless-route-http.test" },
             service   = ngx.null,
           },
+          {
+            paths      = { "/disabled-service1" },
+            protocols  = { "http" },
+            strip_path = false,
+            service    = {
+              path     = "/disabled-service-path/",
+              enabled  = false,
```

Comment:
If the C* migration suggestion is not added then a test should be added to handle the `nil` case for the `enabled` service configuration.